docs/source/reexecuting_updating_experiments.rst

```diff
@@ -0,0 +1,134 @@
+Re-executing and updating experiments
+=====================================
+
+*This tutorial covers updating and re-executing individual cases in an experiment.*
+
+Filtered case execution
+***********************
+
+It is possible to update and run a subset of cases in a batch simulation. Call the execute method on the 
+experiment object with the ``with_cases`` argument set to a list of the cases that should be executed. 
+Observe that if the experiment has not been executed yet it will not have any cases that we can access. 
+To work around this we can first execute the experiment and specify that no cases should be executed::
+
+    experiment_expand = experiment.execute(with_cases=[]).wait()
```

Comment:
Maybe add the code line 
experiment = workspace.create_experiment(experiment_definition)

to make it more clear that you run this just after creating the experiment.